app/src/main/java/org/thoughtcrime/securesms/components/ProfilePictureView.kt

```diff
@@ -53,19 +53,19 @@ class ProfilePictureView @JvmOverloads constructor(
 
     fun update(recipient: Recipient) {
         this.recipient = recipient
-        recipient.run { update(address, isClosedGroupRecipient, isOpenGroupInboxRecipient) }
+        recipient.run { update(address, isLegacyGroupRecipient, isCommunityInboxRecipient) }
     }
 
     fun update(
         address: Address,
-        isClosedGroupRecipient: Boolean = false,
-        isOpenGroupInboxRecipient: Boolean = false
+        isLegacyGroupRecipient: Boolean = false,
+        isCommunityInboxRecipient: Boolean = false
     ) {
         fun getUserDisplayName(publicKey: String): String = prefs.takeIf { userPublicKey == publicKey }?.getProfileName()
             ?: DatabaseComponent.get(context).sessionContactDatabase().getContactWithAccountID(publicKey)?.displayName(Contact.ContactContext.REGULAR)
             ?: publicKey
 
-        if (isClosedGroupRecipient) {
+        if (isLegacyGroupRecipient) {
```

Comment:
What about GroupsV2? Don't they need a specific behaviour too?

Reply:
Used the same logic